ethereum/migrations/prod/10_pyth_make_interface_simpler.js

```diff
@@ -0,0 +1,34 @@
+require('dotenv').config({ path: "../.env" });
```

Comment:
what's the difference between the migrations in `prod` and `prod-receiver`? (you may have explained this to me already but i've forgotten)

Reply:
nevermind. i read the README and it explains it, so 👍